src/util/sandbox/fd_sandbox.h

```diff
@@ -0,0 +1,43 @@
+#ifndef HEADER_fd_src_util_sandbox_fd_sandbox_h
+#define HEADER_fd_src_util_sandbox_fd_sandbox_h
+
+#include "../log/fd_log.h"
+
+enum fd_sandbox_profile {
+  FD_SANDBOX_PROFILE_COMMON,
+  FD_SANDBOX_PROFILE_XDP
```

Comment:
This is apparently not used anywhere?

Reply:
Oh yes, true! We need to talk about. We might need to have a special mode for the quic tile when the driver operates in skb mode. The current reference is inaccurate and I'll just remove it for now.